mobile/lib/providers/authentication.provider.dart

```diff
@@ -165,57 +164,56 @@ class AuthenticationNotifier extends StateNotifier<AuthenticationState> {
         Store.tryGet(StoreKey.deviceId) ?? await FlutterUdid.consistentUdid;
 
     bool shouldChangePassword = false;
-    User? user;
+    User? user = Store.tryGet(StoreKey.currentUser);
 
-    bool retResult = false;
-    User? offlineUser = Store.tryGet(StoreKey.currentUser);
-
-    // If the user is offline and there is a user saved on the device,
-    // if not try an online login
-    if (offlineLogin && offlineUser != null) {
-      user = offlineUser;
-      retResult = false;
-    } else {
-      UserAdminResponseDto? userResponseDto;
-      UserPreferencesResponseDto? userPreferences;
-      try {
-        userResponseDto = await _apiService.usersApi.getMyUser();
-        userPreferences = await _apiService.usersApi.getMyPreferences();
-      } on ApiException catch (error, stackTrace) {
-        _log.severe(
-          "Error getting user information from the server [API EXCEPTION]",
-          error,
-          stackTrace,
-        );
-        if (error.innerException is SocketException) {
-          state = state.copyWith(isAuthenticated: true);
-        }
-      } catch (error, stackTrace) {
-        _log.severe(
-          "Error getting user information from the server [CATCH ALL]",
-          error,
-          stackTrace,
-        );
+    UserAdminResponseDto? userResponse;
```

Comment:
Make final

Reply:
This variable will be re-assigned and checked for null.